ete4/coretype/tree.pyx

```diff
@@ -43,12 +43,10 @@ class TreeError(Exception):
 
 cdef class Tree(object):
     cdef public dict props
-    cdef public set features
     cdef public list _children
     cdef public Tree up
     cdef public object _img_style
     cdef public object _sm_style
-    cdef public object _faces
```

Comment:
This is now in `node.props['_faces']`. only used by treeview module, not smartview